src/treemodel/node.js

```diff
@@ -188,6 +188,9 @@ export default class Node {
 		// Due to circular references we need to remove parent reference.
 		json.parent = this.parent ? this.parent.name : null;
 
+		// Serialize attributes as Map object is represented as "{}" when parsing to JSON.
+		json._attrs = [ ...json._attrs ];
```

Comment:
First of all, `_attrs` is an invalid name, because we should not use such shorthand for a property name (there's a ticket to fix this across the API). Second, it should not be prefixed with underscore, because it's not private in that JSON (cause it's not an API, just data).

tl;dr – rename it to `attributes`.